openmdao/core/problem.py

```diff
@@ -666,7 +666,7 @@ def record_iteration(self, case_name):
             Name used to identify this Problem case.
         """
         warn_deprecation("'Problem.record_iteration' has been deprecated. "
-                         "Use 'Problem.record_state' instead.")
+                         "Use 'Problem.record' instead.")
```

Comment:
Should we deprecate record_state (and add a test for that?)

Reply:
I thought about that but record_state has only been out there for a while and was never released. I can easily add it, obviously!

Reply:
Oh, if it's not in a release then maybe we don't need to.